src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -25,8 +25,7 @@
  *
  * The onKernelResponse method must be connected to the kernel.response event.
  *
- * The WDT is only injected on well-formed HTML (with a proper </body> tag).
- * This means that the WDT is never included in sub-requests or ESI requests.
+ * The WDT is only injected on well-formed HTML (with a proper <title> tag).
```

Comment:
>  We don't want to inject the toolbar if we are not really sure that this is a full HTML page. That's the case now as a `</body>` cannot be part of an HTML fragment. `<title>`
can be used in a fragment in which case it would incorrectly inject the toolbar.

From https://github.com/symfony/symfony/pull/20343#issuecomment-257871750 made me understand that checking for `<title>` has been rejected too, but maybe I missed something.